<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modbus Joystick Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="joystickFrame">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout" name="joystickLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="joystickFrame_2">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton0">
               <property name="text">
                <string>Button 0 (&amp;A)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton1">
               <property name="text">
                <string>Button 1 (&amp;S)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>Index</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Description</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ledsFrame">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout" name="ledsLayout">
         <item>
          <widget class="QLabel" name="binaryLedsLabel">
           <property name="text">
            <string>Binary LEDs</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="binaryLedRed">
           <property name="styleSheet">
            <string>background-color: red;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="binaryLedGreen">
           <property name="styleSheet">
            <string>background-color: green;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="binaryLedBlue">
           <property name="styleSheet">
            <string>background-color: blue;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pwmLedsLabel">
           <property name="text">
            <string>PWM LEDs</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="pwmLedRed">
           <property name="styleSheet">
            <string>background-color: red;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="pwmLedGreen">
           <property name="styleSheet">
            <string>background-color: green;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="pwmLedBlue">
           <property name="styleSheet">
            <string>background-color: blue;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string>dfsdfsdf</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
